Source/WebKit/WPEPlatform/wpe/wayland/WPEViewWayland.cpp

```diff
@@ -384,8 +384,10 @@ static const struct wl_surface_listener surfaceListener = {
         }
         view->priv->monitors.append(monitor);
         wpeViewWaylandUpdateScale(view);
-        if (monitorChanged)
+        if (monitorChanged) {
+            wpe_view_map(WPE_VIEW(view));
```

Comment:
Don't we need to unmap the "previously" mapped view?

Reply:
We don't support multiple view yet, so there's no previously mapped view.